packages/i18n-calypso/README.md

```diff
@@ -341,7 +341,7 @@ export default function Header() {
 }
 ```
 
-# `withRtl` Higher-Order Component
+## `withRtl` Higher-Order Component
```

Comment:
Sorry for commenting here, but GH's interface doesn't allow otherwise. Should the first `## Usage` in this document be `### Usage` instead? If that's true, let's check the rest of the document.

Reply:
You are right, it should be `### Usage`, and this header should be `### withRtl...`.

After those changes, this is the TOC of the file:
```
- [I18n Calypso](#i18n-calypso)
	- [Translate Method](#translate-method)
		- [Options](#options)
		- [Usage](#usage)
		- [Strings Only](#strings-only)
		- [String Substitution](#string-substitution)
		- [Mixing Strings And Markup](#mixing-strings-and-markup)
		- [Pluralization](#pluralization)
		- [More translate() Examples](#more-translate-examples)
	- [numberFormat Method](#numberformat-method)
		- [Examples](#examples)
	- [hasTranslation Method](#hastranslation-method)
		- [Usage](#usage-1)
	- [Mixin](#mixin)
	- [Localize](#localize)
		- [Usage](#usage-2)
	- [React Hook](#react-hook)
		- [Usage](#usage-3)
	- [React Localization Helpers for RTL](#react-localization-helpers-for-rtl)
		- [`useRtl` React Hook](#usertl-react-hook)
		- [`withRtl` Higher-Order Component](#withrtl-higher-order-component)
	- [Some Background](#some-background)
		- [Key Hashing](#key-hashing)
			- [Example](#example)
	- [Extracting Translatable Strings From JavaScript Sources](#extracting-translatable-strings-from-javascript-sources)
```